.github/CODEOWNERS

```diff
@@ -38,7 +38,7 @@
 #/avm/res/databricks/workspace/ @Azure/avm-res-databricks-workspace-module-owners-bicep @Azure/avm-core-team-technical-bicep
 /avm/res/data-factory/factory/ @Azure/avm-res-datafactory-factory-module-owners-bicep @Azure/avm-core-team-technical-bicep
 #/avm/res/data-protection/backup-vault/ @Azure/avm-res-dataprotection-backupvault-module-owners-bicep @Azure/avm-core-team-technical-bicep
-#/avm/res/db-for-my-sql/flexible-server/ @Azure/avm-res-dbformysql-flexibleserver-module-owners-bicep @Azure/avm-core-team-technical-bicep
+/avm/res/db-for-my-sql/flexible-server/ @Azure/avm-res-dbformysql-flexibleserver-module-owners-bicep @Azure/avm-core-team-technical-bicep
```

Comment:
Please ref to https://azure.github.io/Azure-Verified-Modules/specs/shared/#team-memberships

Reply:
@Azure/avm-res-dbformysql-flexibleserver-module-owners-bicep
@Azure/avm-res-dbformysql-flexibleserver-module-contributors-bicep